src/higherOrderComponents/withBannersContext.tsx

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import { BannerContextProvider } from "context/banners";
+
+export const withBannersContext = <T extends unknown>(
+  Component: React.ComponentType<T>
+) => {
+  return (props: T) => (
+    <BannerContextProvider>
+      <Component {...props} />
+    </BannerContextProvider>
+  );
+};
```

Comment:
i think this being shown as added from this PR is a mistake. you might have to rebase